modules/account-lib/src/coin/eos/DeleteAuthActionBuilder.ts

```diff
@@ -0,0 +1,89 @@
+import * as EosJs from 'eosjs';
```

Comment:
Can you please link the documentation for these actions from EOS site? These can be included in each action documentation. It is not easy to find the documentation on EOS site?